### PR TITLE
Display languages on student profile and problems page

### PR DESCRIPTION
**Pull request by DavieV**
Languages were added as a field in the solution model, but they weren't being used in all relevant places.

### CODE REVIEW

File: templates/student/student.html [hunk 1]
```diff
@@ -14,13 +14,16 @@
 <hr />
 
 <table class="table table-striped table-bordered">
-    {% for sol in solutions %}
+    {% for sol in solutions|dictsort:"programming_language" %}
```
Comment (Quinny) [inline templates/student/student.html]: I think we shouldn't dict sort the solutions on the student page.  Just show them in order of the week.

**Review by Quinny — CHANGES_REQUESTED** [review]
Just fix the profile sorting then LGTM.

**Comment by DavieV** [comment]
Done.

**Review by Quinny — APPROVED** [review]

**Comment by Quinny** [comment]
Changes have been pushed to prod.

**Pull request merged**